lib/internal/assert/assertion_error.js

```diff
@@ -41,269 +37,77 @@ const kReadableOperator = {
-      // not a tty, use a default value of 80 characters.
-      const maxLength = process.stderr.isTTY ? process.stderr.columns : 80;
-      if (inputLength < maxLength) {
-        while (actualRaw[i] === expectedRaw[i]) {
-          i++;
-        }
-        // Ignore the first characters.
-        if (i > 2) {
-          // Add position indicator for the first mismatch in case it is a
-          // single line and the input length is less than the column length.
-          indicator = `\n  ${StringPrototypeRepeat(' ', i)}^`;
-          i = 0;
-        }
-      }
-    }
-  }
-
-  // Remove all ending lines that match (this optimizes the output for
-  // readability by reducing the number of total changed lines).
-  let a = actualLines[actualLines.length - 1];
-  let b = expectedLines[expectedLines.length - 1];
-  while (a === b) {
-    if (i++ < 3) {
-      end = `\n  ${a}${end}`;
-    } else {
-      other = a;
-    }
-    ArrayPrototypePop(actualLines);
-    ArrayPrototypePop(expectedLines);
-    if (actualLines.length === 0 || expectedLines.length === 0)
-      break;
-    a = actualLines[actualLines.length - 1];
-    b = expectedLines[expectedLines.length - 1];
-  }
-
-  const maxLines = MathMax(actualLines.length, expectedLines.length);
-  // Strict equal with identical objects that are not identical by reference.
-  // E.g., assert.deepStrictEqual({ a: Symbol() }, { a: Symbol() })
-  if (maxLines === 0) {
-    // We have to get the result again. The lines were all removed before.
-    const actualLines = StringPrototypeSplit(actualInspected, '\n');
-
-    // Only remove lines in case it makes sense to collapse those.
-    // TODO: Accept env to always show the full error.
-    if (actualLines.length > 50) {
-      actualLines[46] = `${colors.blue}...${colors.white}`;
-      while (actualLines.length > 47) {
-        ArrayPrototypePop(actualLines);
-      }
-    }
-
-    return `${kReadableOperator.notIdentical}\n\n` +
-           `${ArrayPrototypeJoin(actualLines, '\n')}\n`;
-  }
-
-  // There were at least five identical lines at the end. Mark a couple of
-  // skipped.
-  if (i >= 5) {
-    end = `\n${colors.blue}...${colors.white}${end}`;
-    skipped = true;
-  }
-  if (other !== '') {
-    end = `\n  ${other}${end}`;
-    other = '';
-  }
+  return operator;
+}
 
-  let printedLines = 0;
-  let identical = 0;
-  const msg = kReadableOperator[operator] +
-        `\n${colors.green}+ actual${colors.white} ${colors.red}- expected${colors.white}`;
-  const skippedMsg = ` ${colors.blue}...${colors.white} Lines skipped`;
+function createErrDiff(actual, expected, operator) {
+  operator = checkOperator(actual, expected, operator);
+  const nopSkippedMessage = `\n${colors.blue}...${colors.white} Lines skipped which didn't differ`;
+  const insertedSkippedMessage = `\n${colors.green}...${colors.white} Lines skipped which were inserted`;
+  const deletedSkippedMessage = `\n${colors.red}...${colors.white} Lines skipped which were deleted`;
```

Comment:
```suggestion
  const nopSkippedMessage = `\n${colors.blue}...${colors.white} Skipped unchanged lines`;
  const insertedSkippedMessage = `\n${colors.green}...${colors.white} Skipped added lines`;
  const deletedSkippedMessage = `\n${colors.red}...${colors.white} Skipped removed lines`;
```

I'm not sure which wording is better. WDYT other reviewers?